presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -283,6 +284,14 @@ primaryExpression
     | '(' expression ')'                                                             #parenthesizedExpression
     ;
 
+mapKey
```

Comment:
Is there a cleaner way to alias these so that I can still separate them when building the AST, without introducing some trivial rules?